src/moh_niger_chw/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="app_name">RCOM</string>
```

Comment:
Name of the app according to the GH issue is:
`RCOM (Relais Communautaire)`

According to the docs linked, the app name is:
`RCom ( Relais Communautaire )`

Please confirm that `RCOM` is what we need.

Reply:
cc @hiellavis

Reply:
Hi @binokaryg 
The name of the application is `RCOM (Relais Communautaire)`.
Thx